backend/routes/submissions.js

```diff
@@ -2,7 +2,7 @@ const express = require("express");
 
 const router = express.Router();
 const db = require("../mongo.js");
-const ObjectId = require("mongodb").ObjectID;
+//const ObjectId = require("mongodb").ObjectID;
```

Comment:
Intentional?

Reply:
Yes. Linter error but will be used later

Reply:
Cool, sounds good to me.